spec/controllers/application_controller_spec.rb

```diff
@@ -190,46 +190,6 @@ def index
     end
   end
 
-  describe '#confirm_user_is_not_suspended' do
-    controller do
-      before_action :confirm_user_is_not_suspended
-
-      def index
-        render plain: 'Hello'
-      end
-    end
-
-    context 'when user is suspended' do
-      it 'redirects to users please call page' do
-        user = create(:user, :suspended)
-        sign_in user
-        get :index
-
-        expect(response).to redirect_to user_please_call_url
-      end
-    end
-  end
-
-  describe '#user_suspended_url' do
```

Comment:
I don't like just removing this, and I would prefer if `after_sign_in_path_for` and `signed_in_url` had test coverage, but they're currently implemented as private methods. Maybe they should be public?

Reply:
This is out of date now right? Or are we still removing this

Reply:
This is up-to-date. I bundled in my suggestion from https://github.com/18F/identity-idp/pull/9147#discussion_r1317725001 which moves `confirm_user_is_not_suspended` and removes `user_suspended_url`.

Reply:
FWIW the accounts controller already has a spec to check for the redirect which I think is better / sufficient for what we were testing here with `confirm_user_is_not_suspended` 🤷 

https://github.com/18F/identity-idp/blob/1c214418c13d7eda9adf2cccc4ffe7bd1e0ca1ec/spec/controllers/accounts_controller_spec.rb#L81-L90

Reply:
Ok that seems fine to me! I didn't realize the method implementation was moved but that makes a ton more sense